source/features/open-all-notifications.tsx

```diff
@@ -69,7 +69,12 @@ function removeOpenUnreadButtons(container: ParentNode = document): void {
 
 function addSelectedButton(selectedActionsGroup: HTMLElement): void {
 	const button = (
-		<button className={'btn btn-sm mr-2 ' + openSelected.class} type="button">
+		<button
+			type="button"
+			className={'btn btn-sm mr-2 tooltipped tooltipped-s ' + openSelected.class}
+			data-hotkey="p"
+			aria-label="Open selected (keyboard shortcut: P)"
```

Comment:
Good tooltip? Other items have the shortcut but no tooltip 🤷‍♂️ but I don't want this shortcut to just exist without anyone knowing. We should try to always make them visible.

Reply:
It might be redundant given the context, but to be explicit, perhaps:

```suggestion
			aria-label="Open selected notification(s) (keyboard shortcut: P)"
```

Reply:
```suggestion
			aria-label="Open selected notifications%A0Hotkey: P"
```

Should appear on two lines